### DISCUSSION TITLE
ebusd esp with Home Assistant and MQTT

### DISCUSSION THREAD

**Discussion by sven055** [john30/ebusd#839] [Q&A]
Hi,
I just installed my ebusd wifi adapter on my Vaillant ecoTec plus Heater with the multimatic VRC700 and installed the ebusd add-on within Home Assistant. I also copied the 15.700.csv and the mqtt-hassio.cfg to the right config path in Home Assistant. But unfortunately there is no data flow with useable values in the log file of the ebusd add-on in Home Assistant. Possibly someone can help me to find a way getting data of my system. 

**eBUSd log Home Assistant**
 
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
> ebusd --foreground --mqtthost=core-mosquitto --mqttport=1883 --mqttuser=addons --mqttpass=uW9eexe4aidi9ik6Fae6eo8gai6ophied5jiekohThei4chithah1ahs7Chekaed --scanconfig --mqttjson --configpath=/config/ebusd/ --latency=10 --pollinterval=5 --mqttint=/config/ebusd/ --mqtttopic=ebus-vaillant --device=192.168.178.47:9999 --log=all:notice --log=bus:info
2023-02-12 14:58:36.548 [main notice] ebusd 23.1.p20230106 started with auto scan on device 192.168.178.47:9999
2023-02-12 14:58:36.550 [main error] error reading config files from /config/ebusd/: ERR: element not found, last error: 15.700.csv:7: ERR: element not found, field type TEMPV in field 1
2023-02-12 14:58:36.584 [bus notice] bus started with own address 31/36
2023-02-12 14:58:37.084 [mqtt notice] connection established
2023-02-12 15:00:46.896 [main notice] update check: revision 23.1 available

**ebusd wifi adapter**

Build: 20221215
ebusd device string: ens:192.168.178.47:9999
WiFi signal: 48% (-76dBm)
Chip ID: 0038951a, [d1mini@80](http://192.168.178.47/?verbose=)
Hostname: ebus-vaillant
Up time: 623
Free heap: 24512
ebusd connected: yes (inactive)
eBUS signal: acquired

14:25:35.9: wifi evt 8
14:25:37.2: wifi evt 2
14:25:40.9: wifi evt 0
14:25:40.9: wifi evt 3
14:25:43.0: check eBUS conn
14:25:43.4: check eBUS conn: 0
14:25:43.4: eBUS >SYN enh
14:25:43.4: skip
14:25:43.4: skip first 1
14:25:43.4: setup: error=0, wifi=1, ip=1
14:25:44.1: serial <f0
14:25:44.4: serial <80
14:25:48.4: start
14:25:51.0: conn 192.168.178.63:41726
14:26:36.9: disconn
14:26:36.9: conn 192.168.178.54:35518
14:43:03.6: disconn
14:43:39.1: conn 192.168.178.54:37304
14:51:30.2: disconn
14:51:37.2: conn 192.168.178.54:38108

**MQTT-Explorer**
ebus-vaillant
global
version = "ebusd 23.1.p20230106"
running = true
scan = "finished"
signal = true
updatecheck = "revision 23.1 available"
uptime = 960

**Comment by Koky05** [comment]
You also need _templates.csv file and some others to run it correctly.
Now it could not decode it because definition of units is missing:
`error reading config files from /config/ebusd/: ERR: element not found, last error: 15.700.csv:7: ERR: element not found, field type TEMPV in field 1`
